bai-tap-buoi-13/main.js

```diff
@@ -0,0 +1,126 @@
+// Exercise 1:
+const getSum = (first, second) => {
+  let sum = 0
+  first = document.getElementById('first')
+  second = document.getElementById('second')
+  sum = parseInt(first.value) + parseInt(second.value)
+  document.getElementById('result').innerHTML = 'Tổng = ' + sum
+}
+
+// Exercise 2:
+const handleOnMouseClick = event => {
+  const currentEle = event.target
+  if (currentEle.style.background === 'red') {
+    currentEle.style.background = 'green'
+  } else {
+    currentEle.style.background = 'red'
+  }
+}
+
+// Exercise 3:
+const changedOption = () => {
+  let value = document.getElementById("electric").value
+  if (value === 'laptop') {
+    document.getElementById('showOptionLaptop').style.display = 'block'
+    document.getElementById('showOptionPhone').style.display = 'none'
+    document.getElementById('showOptionTV').style.display = 'none'
+  }
+  else if (value === 'phone') {
```

Comment:
đặt ngoặc kiểu gì đây?
```suggestion
  } else if (value === 'phone') {
```